tools/webcam/configs/gesture/try_det.py

```diff
@@ -0,0 +1,104 @@
+# Copyright (c) OpenMMLab. All rights reserved.
```

Comment:
Is this file really needed?

Reply:
No. It will be replaced in the coming commit with a clean config.